src/usdb_syncer/notes_parser.py

```diff
@@ -401,6 +401,22 @@ def sanitize(self) -> None:
         self.headers.reset_file_location_headers()
         self.notes.maybe_split_duet_notes()
         self.restore_missing_headers()
+        if self.headers.language == "Polish":
+            self.fix_polish_song()
+
+    def fix_polish_song(self) -> None:
+        self.headers.artist = _fix_polish_letters(self.headers.artist)
+        self.headers.title = _fix_polish_letters(self.headers.title)
+        # edition, genre, creator, p1, p2, album, comment, anything else?
```

Comment:
I guess we could just iterate over all headers indiscriminately, but I'd have to investigate how that would work myself. Unless you want to do it yourself, I'll have a look later.

Reply:
I tried to iterate over the respective headers with something like `for tag in [self.headers.artist, self.headers.title, ...]: tag = _fix_polish_letters(tag)`, but that didn’t work.